### ISSUE TITLE
Any guidance on creating smaller images? 256x256 or 384x384?

### ISSUE THREAD

**Issue by MatthewWaller** [huggingface/diffusers#953]
**Is your feature request related to a problem? Please describe.**
I’m looking for ways to speed the process and save memory

**Describe the solution you'd like**
I wondered if it is possible to create smaller images.

**Describe alternatives you've considered**
I tried setting the images size with poor results out of the box. Thoughts?

**Comment by Lewington-pitsos** [comment]
I spent like 60 minutes googling this yesterday, I checked a bunch of different forums and the consensus seems to be that due to the fact that SD was trained on 512x512 resolution, lower resolutions will not work. 

To achieve this I think you would have to re-train or at least read the code and work out something that's not very obvious. I can confirm that there are no guides on doing this at present.

One sneaky trick I tried was "four separate images of a forest" and it generated like 4 images in a 2x2 grid, but that's crazy janky.
